packages/theming/src/elements/ThemeProvider.tsx

```diff
@@ -20,7 +20,7 @@ export const ThemeProvider = ({
   ...other
 }: PropsWithChildren<IThemeProviderProps>) => {
```

Comment:
Ultimately, I think we'll expect to see [createGlobalStyle](https://styled-components.com/docs/api#createglobalstyle) here with a listing of CSS variables pulled from the default theme.